### ISSUE TITLE
Keyboard shortcuts not preserved after restart

### ISSUE THREAD

**Issue by JanJakes** [warpdotdev/Warp#801]
### Describe the bug

```shell
I set up some keyboard shortcuts for tab switching on non-English keyboard.
It works but after restarting Warp these shortcut settings are not preserved.
```

### To Reproduce

Steps to reproduce:
1. Go to `Settings > Keyboard shortcuts`.
2. Set `Switch to 1st tab` to some custom shortcut.
3. Restart Warp.
4. Go back to the settings and the shortcut is unchanged.

### Expected behaviour

```shell
The shortcut should be preserved after Warp restart.
```

### Screenshots

This is what I've set up and what doesn't get preserved after Warp restart:
![Screen Shot 2022-03-10 at 10 46 04](https://user-images.githubusercontent.com/141436/157635479-9ec30f9a-9567-475c-9330-132ee3603714.png)

### Operating System

MacOS

### OS Version

12.2.1

**Comment by szgupta** [comment]
Hi @JanJakes, thanks for reporting this! We persist the keybindings in the `~/.warp` folder and so I think we aren't able to persist them in this case because such a folder doesn't exist. We're working on a fix to create this directory for users if they don't already have it. 

In the meantime though, can you try creating this folder manually (i.e. `mkdir ~/.warp`) before editing and saving your custom keybindings?

**Comment by JanJakes** [comment]
Thanks, @szgupta! Indeed, I had no `~/.warp` folder and after I created it Warp does save the bindings into `~/.warp/keybindings.yaml` and it works. Thanks!

**Comment by szgupta** [comment]
No problem! this should be fixed in the next release :)

**Comment by szgupta** [comment]
Closing since this was fixed in the latest release `v0.2022.03.14.08.49.stable_00`!
